ImportExport/CadbaseLibrary/CdbsApi.py

```diff
@@ -0,0 +1,51 @@
+''' Functionality for processing requests to the CADBase platform '''
+
+import json
+import pathlib
+from PySide import QtCore
```

Comment:
Please warn that `PySide` is not the expected `PySide` with
```
from PySide import QtCore  # FreeCAD's PySide
```

I don't like that FreeCAD "steals" `PySide` but that's how it is. Most probably, `PySide` imported from within FreeCAD is `PySide2` but do not mix them anyway, stick to `PySide`.